client/src/pages/track/Track.svelte

```diff
@@ -31,7 +34,7 @@
                 origin: first.target === null ? null : allOffices[first.target] ?? null,
                 current: last.target === null ? null : allOffices[last.target] ?? null,
             };
-
+        }
```

Comment:
Let's add a newline here so that it's less cramped.
```suggestion
        }

```